text/code-standards/formatting.md

```diff
@@ -0,0 +1,20 @@
+# Formatting
+
+All Rust projects should follow the same formatting standard. This standard is
+to be documented in a `rustfmt.toml` file in the top level of each project.
+See below for an example of the default values for the file.
+
+```toml
```

Comment:
Another nice one for organizing imports is [group_imports](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports)

```suggestion
```toml
group_imports = "StdExternalCrate"
```